airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json

```diff
@@ -105,6 +105,12 @@
             }
           }
         ]
+      },
+      "custom_reports": {
```

Comment:
Please update our internal secrets config for this connector with this field set to ensure SAT run is testing this stream with data.

Reply:
Done by defining jsonschema

Reply:
Updated